Wrappers/Python/test/test_reconstructors.py

```diff
@@ -36,8 +36,12 @@
 
 has_tigre_gpu = has_gpu_tigre()
 if  not has_tigre_gpu:
-    print("Unable to run TIGRE tests")
+    print("Unable to run ASTRA tests")
```

Comment:
Why is the message Unable to run ASTRA tests? Shouldn't it be TIGRE?

Reply:
Probably :D